src/common/collection.c

```diff
@@ -735,61 +569,104 @@ static void _collection_update_aspect_ratio(const dt_collection_t *collection)
+      return _("id");
+    case DT_COLLECTION_SORT_COLOR:
+      return _("color label");
+    case DT_COLLECTION_SORT_GROUP:
+      return _("group");
+    case DT_COLLECTION_SORT_PATH:
+      return _("full path");
+    case DT_COLLECTION_SORT_CUSTOM_ORDER:
+      return _("custom sort");
+    case DT_COLLECTION_SORT_TITLE:
+      return _("title");
+    case DT_COLLECTION_SORT_DESCRIPTION:
+      return _("description");
+    case DT_COLLECTION_SORT_ASPECT_RATIO:
+      return _("aspect ratio");
+    case DT_COLLECTION_SORT_SHUFFLE:
+      return _("shuffle");
+    default:
+      return "";
   }
-  if(reverse != -1) params->descending = reverse;
-
-  _collection_update_aspect_ratio(collection);
-}
-
-dt_collection_sort_t dt_collection_get_sort_field(const dt_collection_t *collection)
-{
-  return collection->params.sort;
-}
+};
 
-gboolean dt_collection_get_sort_descending(const dt_collection_t *collection)
-{
-  return collection->params.descending;
-}
-
-const char *dt_collection_name(dt_collection_properties_t prop)
+const char *dt_collection_name_untranslated(dt_collection_properties_t prop)
 {
   char *col_name = NULL;
   switch(prop)
   {
-    case DT_COLLECTION_PROP_FILMROLL:         return _("film roll");
-    case DT_COLLECTION_PROP_FOLDERS:          return _("folder");
-    case DT_COLLECTION_PROP_CAMERA:           return _("camera");
-    case DT_COLLECTION_PROP_TAG:              return _("tag");
-    case DT_COLLECTION_PROP_DAY:              return _("date taken");
-    case DT_COLLECTION_PROP_TIME:             return _("date-time taken");
-    case DT_COLLECTION_PROP_IMPORT_TIMESTAMP: return _("import timestamp");
-    case DT_COLLECTION_PROP_CHANGE_TIMESTAMP: return _("change timestamp");
-    case DT_COLLECTION_PROP_EXPORT_TIMESTAMP: return _("export timestamp");
-    case DT_COLLECTION_PROP_PRINT_TIMESTAMP:  return _("print timestamp");
-    case DT_COLLECTION_PROP_HISTORY:          return _("history");
-    case DT_COLLECTION_PROP_COLORLABEL:       return _("color label");
-    case DT_COLLECTION_PROP_LENS:             return _("lens");
-    case DT_COLLECTION_PROP_FOCAL_LENGTH:     return _("focal length");
-    case DT_COLLECTION_PROP_ISO:              return _("ISO");
-    case DT_COLLECTION_PROP_APERTURE:         return _("aperture");
-    case DT_COLLECTION_PROP_EXPOSURE:         return _("exposure");
-    case DT_COLLECTION_PROP_ASPECT_RATIO:     return _("aspect ratio");
-    case DT_COLLECTION_PROP_FILENAME:         return _("filename");
-    case DT_COLLECTION_PROP_GEOTAGGING:       return _("geotagging");
-    case DT_COLLECTION_PROP_GROUPING:         return _("grouping");
-    case DT_COLLECTION_PROP_LOCAL_COPY:       return _("local copy");
-    case DT_COLLECTION_PROP_MODULE:           return _("module");
-    case DT_COLLECTION_PROP_ORDER:            return _("module order");
-    case DT_COLLECTION_PROP_RATING:           return _("rating");
+    case DT_COLLECTION_PROP_FILMROLL:
+      return "film roll";
```

Comment:
Ok, not translated, I'm sure you have good reasons. Still I think it should be recorded for translation so using:

`  N_("...")`